components/admin/reports/GenerateReportModal.tsx

```diff
@@ -39,6 +47,63 @@ export default function GenerateReportModal<T>(props: GenerateReportProps<T>) {
   const [endDate, setEndDate] = useState('');
   const [step, setStep] = useState(GenerateReportStep.SelectColumns);
 
+  // Checkbox states
+  const [selectAll, setSelectAll] = useState(false);
+  const [userID, setUserID] = useState(false);
+  const [applicantName, setApplicantName] = useState(false);
+  const [applicantDoB, setApplicantDoB] = useState(false);
+  const [APPNumber, setAPPNumber] = useState(false);
+  const [applicationDate, setApplicationDate] = useState(false);
+  const [paymentMethod, setPaymentMethod] = useState(false);
+  const [feeAmount, setFeeAmount] = useState(false);
+  const [donationAmount, setDonationAmount] = useState(false);
+  const [totalAmount, setTotalAmount] = useState(false);
+
+  // Toast message
+  const toast = useToast();
+
+  // Export csv query
+  const [exportCSV, { loading }] = useLazyQuery<
+    GenerateApplicantsReportResponse,
+    GenerateApplicantsReportRequest
+  >(GENERATE_APPLICANTS_REPORT_QUERY, {
+    onError: error => {
```

Comment:
Can we add a simple `onCompleted` and show a success toast?

Reply:
Done! I added `A CSV report has been successfully generated` as the displayed toast descriotion. I'll message the designers to see if they want a different message

Reply:
Sounds good, thanks!